pepper-apis/src/main/java/org/broadinstitute/ddp/DataDonationPlatform.java

```diff
@@ -629,43 +636,47 @@ private static void setupApiActivityFilter() {
         });
     }
 
-    private static void setupMDC(String path, Route route) {
-        pathToClass.put(path, route.getClass().getSimpleName());
-        before(path, (request, response) -> MDC.put(MDC_ROUTE_CLASS, pathToClass.get(path)));
+    private static void setupMDC(String path, Route route, String operation) {
+        pathToClass.put(operationAndPath(operation, path), route.getClass().getSimpleName());
+        before(path, (request, response) -> MDC.put(MDC_ROUTE_CLASS, pathToClass.get(operationAndPath(request.requestMethod(), path))));
+    }
+
+    private static String operationAndPath(String operation, String path) {
+        return operation + '_' + path;
```

Comment:
IMO this operation just a few characters and not used anywhere else, so don't see the need need of a method for this.

Reply:
Removed